seacrowd/utils/schemas/chat.py

```diff
@@ -0,0 +1,24 @@
+"""
+Conversational Chat Schema
+"""
+import datasets
+
+features = datasets.Features(
+    {
+        "id": datasets.Value("string"),
+        "input": datasets.Sequence({
+            "role": datasets.ClassLabel(names=["system", "user", "assistant"]),
```

Comment:
the role should be implemented as `string` instead of `ClassLabel` due to 2 reasons:
1. There might be some cases where the role can be multiple for a single-sequence dialogues
2. The actual data (when the examples are generated) will result in an integer instead of string